src/sugar3/graphics/toolbutton.py

```diff
@@ -28,6 +27,12 @@
 from sugar3.graphics.icon import Icon
 from sugar3.graphics.palette import Palette, ToolInvoker
 
+'''
+Toolbutton is a simple button which can contain only
+one event, for example, the clear search button
+in journal.
```

Comment:
The clear search button is just a regualr button.

Toolbuttons are for putting in toolbars.